### DISCUSSION TITLE
Taiko Touch Input

### DISCUSSION THREAD

**Discussion by sa-shiro** [ppy/osu#19425] [Ideas]
Referencing https://github.com/ppy/osu/pull/17200

This is great, but for large phones like the Samsung Z Flip the outer rims are visually too small and the inner drum is a little hard to hit 

I'd suggest either changing or adding this as a control type:

Using the left and right edge sides of the phone for the controls instead of the center (and especially making the outer rim visually a bit bigger, if added as an additional control type) - if this will be changed or added, it can be essentially reused for mania I assume? (at least for 4K)

**Comment by peppy** [comment]
As per the changelog: This is a first pass implementation of touch support for osu!taiko. Eventually we plan to give the user a lot more control over touch input customisation, but hopefully this will suffice until that point.

The drum was made to take up as much of the screen without intercepting the playfield.
